### DISCUSSION TITLE
不小心删除了所有配置，怎么恢复

### DISCUSSION THREAD

**Discussion by 1f2f3f4f** [2dust/v2rayNG#3147] [General]
在提出问题前请先自行排除服务器端问题，同时也请通过搜索确认是否有人提出过相同问题。

### 预期行为
描述你认为应该发生什么

### 实际行为
描述实际发生了什么

### 复现方法
1.
2.
3.

### 日志信息
<details>

通过`adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main`获取日志。请自行删减日志中可能出现的敏感信息。

如果问题可重现，建议先执行`adb logcat -c`清空系统日志再执行上述命令，再操作重现问题。
```
在这里粘贴日志
```
</details>

**Comment by 2dust** [comment]
最新版有备份配置功能
如果你没有使用，则没有办法恢复
